Git Testing/Scenes/Listing/ListingVC.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ViewController.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 14/01/2021.
+//
+
+import UIKit
+
+class ListingVC: UIViewController {
+    
+    //MARK:- IBOUTLETS
+    @IBOutlet var tableView:UITableView!
+    
+    //MARK:- PROPERTIES
+    lazy var viewModel: ListViewModel = {
+        return ListViewModel()
+    }()
```

Comment:
Make the object lazy if it is doing some heavy work at initialization or its not sure whether it will be used or not. Otherwise you can create normal properties if not required for any specific usecase.